imdb/SearchResults.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import sys
+import traceback
+
+import requests
+
+from bs4 import BeautifulSoup
+
+
+def get_search_movies(movie):
+
+    url = "http://www.imdb.com/find?s=tt&q=" + movie
```

Comment:
not a tag name but a query parameter which denotes all the all the movies matching to the parameter q instead of pages